commons-dlp/src/main/java/org/exoplatform/commons/dlp/service/impl/DlpPositiveItemServiceImpl.java

```diff
@@ -64,6 +64,21 @@ public void deleteDlpPositiveItem(Long itemId) {
       LOG.warn("The DlpItem's {} not found.", itemId);
     }
   }
+ 
+   @Override
+  public void restoreDlpPositiveItem(Long itemId) {
+    try {
+      DlpPositiveItemEntity dlpPositiveItemEntity = dlpPositiveItemDAO.find(itemId);
+      if (dlpPositiveItemEntity != null) {
+        dlpPositiveItemDAO.delete(dlpPositiveItemEntity);
+        listenerService.broadcast(new Event("dlp.listener.event.move.document", null, dlpPositiveItemEntity.getReference()));
```

Comment:
```suggestion
        listenerService.broadcast(new Event("dlp.listener.event.restore.item", null, dlpPositiveItemEntity.getReference()));
```
I think we should rename the event